.github/workflows/cloudflare-worker.yml

```diff
@@ -19,19 +19,14 @@ jobs:
           - 14
```

Comment:
might be time to drop node 14 too?

Reply:
gonna leave it in here until we actually drop node 14 support